nemo_skills/prompt/config/generic/gpqa-zero-shot.yaml

```diff
@@ -0,0 +1,6 @@
+# Prompt used by TigerLab for MMLU-Pro benchmark
+# https://github.com/TIGER-AI-Lab/MMLU-Pro/blob/main/evaluate_from_api.py
+
+user: |-
```

Comment:
Does it make a big difference if we use this vs mmlu? Don't mind having an extra prompt if it's important, but if we can unify them, it's always better